x-pack/plugins/monitoring/public/components/cluster/overview/helpers.js

```diff
@@ -42,15 +43,14 @@ export function HealthLabel(props) {
     }
   }
 
-  if (product === 'kb' && status === 'red') {
+  // TODO: Use the actual service level statuses instead of converting them to colors
+  if (product === 'kb' && (status === 'yellow' || status === 'red')) {
     return (
       <EuiText>
         {i18n.translate('xpack.monitoring.cluster.health.pluginIssues', {
-          defaultMessage: 'Some plugins are experiencing issues. Check ',
+          defaultMessage: 'Some plugins may be experiencing issues. Please check ',
         })}
-        <EuiLink href="/status" target="_blank" external={true}>
-          status
-        </EuiLink>
+        <EuiLink href={`${Legacy.shims.getBasePath()}/status`}>the Kibana status page</EuiLink>.
```

Comment:
Guess this might be out of scope for this PR, but it seems like this might be linking back to the current kibana. I would have expected us to be linking to the monitored kibana. Or maybe SM doesn't have that info available?

Reply:
This is a great point :/

Reply:
Gotcha. In that case I suspect this pr is no worse than the current state. Just more to fix for later :)